src/backend/api/meals.js

```diff
@@ -1,15 +1,63 @@
+const knex = require("knex");
 const express = require("express");
-const router = express.Router();
-const knex = require("../database");
+const database = require("../database");
+const app = express();
```

Comment:
No need to create another `express` app here, you already have one in `app.js`. 😄